examples/integration_with_kura.py

```diff
@@ -0,0 +1,259 @@
+                                    raise
+                                relationship_attempts += 1
+                                log.error(
+                                    'Trying Device {0} relationship with Edge System failed with following error - {1}'
+                                        .format(device.name, str(e)))
+                                log.info(
+                                    '{0} Device Relationship: Attempt: {1}'.format(device.name, str(relationship_attempts)))
+                                time.sleep(delay_retries)
+
+                        # Use the device name as identifier in the registry to easily refer the device in other packages
+                        device_registry_name = Jsonpayload['assetName']
+
+                        userdata['registry'].register(device_registry_name, reg_device)
+
+                        # Setting multiple properties by passing Dictonary object for Devices with the retry attempts
+                        # in case of exceptions
+                        prop_attempts = 0
+                        while prop_attempts <= retry_attempts:
+                            try:
+                                userdata['self'].iotcc.set_properties(reg_device,
+                                                          {"Country": "CN-B", "State": "BeiJing", "City": "China",
+                                                           "Location": "VMware BeiJing", "Building": "BeiJing",
+                                                           "Floor": "BeiJing Floor"})
+                                break
+
+                            except Exception as e:
+                                prop_attempts = prop_attempts + 1
+                                log.error('Exception while setting property for Device {0} - {1}'
+                                    .format(device.name, str(e)))
+                                log.info('Trying setting properties for Device {0}: Attempt - {1}'
+                                         .format(device.name,str(prop_attempts)))
+                                time.sleep(delay_retries)
+
+                    except Exception:
+                        log.info("Device Registration and Metrics loading failed")
+                        self.clean_up()
+                        raise
+
+                    Gatewaydict[Jsonpayload['assetName']] = reg_device
+            else:
+                log.info("Jsonpayload empty")
+        else:
+            log.info("another topic")
+
+
+    def run(self, registry):
+        """
+        The execution function of a liota package.
+
+        Acquires "iotcc_mqtt" and "iotcc_mqtt_edge_system" from registry and registers edge_system related metrics
+        with the DCC and publishes those metrics.
+
+        :param registry: the instance of ResourceRegistryPerPackage of the package
+        :return:
+        """
+        self.iotcc = registry.get("iotcc_mqtt")
```

Comment:
Please add detailed comments on how data is getting published to IoTCC after json Parsing.